lib/htmlValidator.js

```diff
@@ -10,6 +10,18 @@ colors.setTheme({
 module.exports = function(app) {
   var params = app.get('params'),
       options,
+      i,
+      errorArr,
```

Comment:
Can you chunk these vars in relevant parts, makes it less glaring and more organized.